spec/jmap/push.mdown

```diff
@@ -26,25 +31,39 @@ Refer to the Authentication section of this spec for details on how to get the U
 
 A client MAY hold open multiple connections to the event-source, although it SHOULD try to use a single connection for efficiency.
 
-## Web hook
+## Callback URL
 
-The second push mechanism is to register a callback URL to which the JMAP server will make an HTTPS POST request whenever the event occurs. The request MUST have a content type of `application/json` and contain the same UTF-8 JSON encoded object as described above as the body.
+The second push mechanism is to register a callback URL to which the JMAP server will make an HTTPS POST request whenever an event occurs. This is compatible with the Application Server role of [@!RFC8030].
```

Comment:
What is compatible?  The URL, the thing using the URL, or the resource that the URL refers to?